src/components/LandingPages/ClientLanding.tsx

```diff
@@ -15,23 +13,67 @@ class ClientLanding extends Component<{}, {}, {}> {
           <title>Home</title>
           <meta name="description" content="Keep.id" />
         </Helmet>
-        <div className="row m-auto mt-5">
-          <div className="d-flex p-2" id="Print container">
-            <Link to="/my-documents">
-              <div className="rectangle pt-2">
-                <img className="normalImage" src={FileSVG} alt="Print" />
-                <p className="textLanding mt-4 pt-3">My Documents</p>
+        <div className="jumbotron-fluid ml-4 mt-2 row">
+          <h1 className="display-4 text-sm-center">Welcome, Name!</h1>
+        </div>
+        <div className="row ml-2 justify-content-center mt-4">
```

Comment:
Is there any way we can reduce the number of divs? It looks like there are a lot of divs and I'm not sure whether all of them are needed. If there are too many divs it might be confusing for other developers to figure out what each div is doing.